server/helpers/reviews_helpers.js

```diff
@@ -1,7 +1,7 @@
 const axios = require('axios');
 const key = require('../config.js');
 const getReviews = () => {
-  return axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/reviews/?product_id=28215', {
+  return axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/reviews/?product_id=29000&count=100', {
```

Comment:
Consider following Salma's example and including the API url in your config file with your authorization key. This way your get and posts could look like this : `axios.get('${key.API}/reviews'. 

This creates more readable code and allows for easily managing urls and keys from config as app size increases.